_data/toc/cloud-guide.yml

```diff
@@ -269,18 +255,16 @@ pages:
 
             - label: Set up Solr service
```

Comment:
Remove all three lines here and anything else where you removed the `includes 2.0` only.